pygerduty/v2.py

```diff
@@ -71,11 +71,38 @@ def create(self, **kwargs):
         if "requester_id" in kwargs:
             extra_headers["From"] = kwargs.pop("requester_id")
 
-        data[self.sname] = kwargs
+        new_kwargs = {}
+
+        for kwarg_key, kwarg_value in kwargs.iteritems():
+            if kwarg_key.endswith('_id'):
+                new_kwargs = Collection.id_to_obj(kwarg_key, kwarg_value)
```

Comment:
Looks like you're overwriting new_kwargs here? Can you move this entire loop into a function and add a test for it to make sure we're not losing kwargs? The test should have kwargs that are and aren't transformed and test both id and ids types.